vivisect/defconfig.py

```diff
@@ -46,6 +46,11 @@
                 'case_failure': 5000,
                 'min_func_instr_size': 10,
                 'timeout_secs': 45,
+            },
+            'codeflow':{
+                'strict':False,
+                'onelib':False,
```

Comment:
So you say false here, but shouldn't it be a value?

Reply:
TODO:  onelib: True to default

Reply:
```suggestion
                'onelib':True,
```